src/routes/courseRoutes.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const { createCourse , getCourse , getAllCourses , updateCourse , deleteCourse  } = require('../controllers/courseController');
+// const { verifyToken } = require('../middlewares/verifyToken');
+
+const router = express.Router();
+
+// route to get all courses 
+router.get('/', getAllCourses);
+
+// route to get course data by uid
+router.get('/:uid', getCourse);
+
+//route for create course
+router.post('/createCourse', createCourse);
+
+// route for Delete Course
+router.delete('/deleteCourse/:uid', deleteCourse);
+
+// route for updating Course 
+router.patch('/updateCourse/:uid', updateCourse);
+
+
+// // route to get all courses 
```

Comment:
If we aren't using this code, we should delete it to avoid being confusing.

Reply:
Added update course function in courseController.js file.